src/include/stat.h

```diff
@@ -121,17 +123,43 @@ __wt_stats_aggregate(void *stats_arg, int slot)
     return (aggr_v);
 }
 
+static WT_INLINE int64_t
+__wt_stats_aggregate_conn(void *stats_arg, int slot)
```

Comment:
Just as a sanity check, do both versions of both of these functions get used? (i.e. do we call both aggregate and clear for both conn and dsrc stats?) If not, we can remove the unused ones and simplify a little bit.

Reply:
Yes they both get used; and for the clear as well.

Reply:
Thanks!